R/pkg/inst/tests/test_rdd.R

```diff
@@ -141,7 +141,8 @@ test_that("PipelinedRDD support actions: cache(), persist(), unpersist(), checkp
   unpersist(rdd2)
   expect_false(rdd2@env$isCached)
 
-  setCheckpointDir(sc, "checkpoints")
+  tempDir <- tempfile(pattern = "checkpoint")
```

Comment:
I am not sure if this is an issue, but does this create a file rather than directory? the checkpoint may not work. Can you unlink the result and make a directory out of it? or can `tempdir()` make a unique subdirectory?